website/content/docs/commands/fmt.mdx

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
I forgot that we need to add this page to the navbar for it to be rendered. If you add this bit of JSON:

```
      {
        "title": "fmt",
        "path": "commands/fmt"
      },
```

between the "commands/eval" and "commands/jobs" at https://github.com/hashicorp/nomad/blob/main/website/data/docs-nav-data.json#L431-L432, that'll allow our preview build for the docs to work.